umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -221,6 +221,19 @@
      (dissoc :ComposedOf)
      (dissoc :NumberOfInstruments)))
 
+(defn migrate-tiling-identification-systems
+  "Migrate TilingIdentificationSystems from 1.9 up to 1.10"
```

Comment:
@mschmele Please just add a comment making it clear that we can drop tiling id systems here.